### ISSUE TITLE
Fix linting errors

### ISSUE THREAD

**Issue by odv** [jscraft-org/mentoring-project-4#10]
When starting this project in Visual Studio, I get the following message:
![error](https://user-images.githubusercontent.com/5067625/31772189-81ae3dee-b4e7-11e7-8456-cc1cd6990523.jpg)

**ESLINT log:**
[Info  - 4:02:25 PM] ESLint server is running.
[Info  - 4:02:26 PM] ESLint library loaded from: /home/ovidiu/Projects/mentoring-project-4/node_modules/eslint/lib/api.js
[Error - 4:02:29 PM] /home/ovidiu/Projects/mentoring-project-4/node_modules/eslint-config-airbnb-base/rules/best-practices.js: 	Configuration for rule "no-else-return" is invalid: 	Value "[object Object]" should NOT have more than 0 items.  Referenced from: /home/ovidiu/Projects/mentoring-project-4/node_modules/eslint-config-airbnb-base/index.js Referenced from: /home/ovidiu/Projects/mentoring-project-4/.eslintrc

**.eslintrc:**
```
{
  "extends": ["airbnb-base", "prettier"],
  "rules": {
    "semi": ["error", "never"],
    "quotes": ["error", "double"]
  }
}
```

**package.json:**
```
{
  "name": "mentoring-project-4",
  "version": "1.0.0",
  "description": "",
  "main": "src/index.js",
  "scripts": {
    "eslint": "eslint 'src/**'"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/jscraft-org/mentoring-project-4.git"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "bugs": {
    "url": "https://github.com/jscraft-org/mentoring-project-4/issues"
  },
  "homepage": "https://github.com/jscraft-org/mentoring-project-4#readme",
  "dependencies": {
    "body-parser": "^1.18.2",
    "dotenv-safe": "^4.0.4",
    "eslint-config-airbnb": "^16.1.0",
    "express": "^4.16.1",
    "mongoose": "^4.12.0",
    "pug": "^2.0.0-rc.4"
  },
  "devDependencies": {
    "eslint": "^4.8.0",
    "eslint-config-airbnb-base": "^12.1.0",
    "eslint-config-prettier": "^2.6.0",
    "nodemon": "^1.12.1"
  }
}
```

I've researched this issue online but I couldn't find a solution. Is this an eslint bug that should be reported?

**Comment by serge-nikitin** [comment]
You should install `eslint-config-airbnb-base` like it described in documentation:

https://www.npmjs.com/package/eslint-config-airbnb-base

`
(
  export PKG=eslint-config-airbnb-base;
  npm info "$PKG@latest" peerDependencies --json | command sed 's/[\{\},]//g ; s/: /@/g' | xargs npm install --save-dev "$PKG@latest"
)
`

so, your devDependencies should have such things:

```
 "eslint": "^4.9.0",
"eslint-config-airbnb-base": "^12.1.0",
"eslint-plugin-import": "^2.8.0"
```

**Comment by odv** [comment]
That was the issue. Need to be more careful when reading documentation.

**Comment by odv** [comment]
I've updated my files in order to fix those linting errors but I still have 3 linting errors that I'm not sure how to fix. I've checked the airBnB style guide but still don't understand why that rule is necessary in my code.

**router.js**
line 39: [eslint] Use object destructuring. (prefer-destructuring)
```const id = req.params.id```

line 57:  [eslint] Use object destructuring. (prefer-destructuring)
```const id = req.params.id```

**Topic.model.js**
line 3: [eslint] Use object destructuring. (prefer-destructuring)
